projects/go-lib/src/lib/components/go-table/go-table.component.scss

```diff
@@ -74,6 +74,14 @@ $go-table-body-row-expanded-border: rgba(240, 240, 240, 0);
     &:last-of-type {
       border-top-right-radius: $global-radius;
     }
+
+    &--sticky {
+      @include z-index(2);
+      background: $theme-light-bg;
+      box-shadow: inset 0 -2px 0 $theme-light-app-bg;
```

Comment:
This doesn't need added to the `--sticky` class since we're replacing it in the base `__head-column` class.